.buildkite/dagster-buildkite/dagster_buildkite/steps/dagster.py

```diff
@@ -433,6 +433,14 @@ def graphql_pg_extra_cmds_fn(_):
     ),
     ModuleBuildSpec("python_modules/libraries/dagster-mlflow", upload_coverage=False),
     ModuleBuildSpec("python_modules/libraries/dagster-mysql", extra_cmds_fn=mysql_extra_cmds_fn),
+    ModuleBuildSpec(
+        "python_modules/libraries/dagster-pandera",
+        supported_pythons=[
```

Comment:
Ideally, this would have the same gating logic around the release branch / master, so that we're just running one python version (e.g. 3.9) instead of all 3.